features/step_definitions/stepdefs.rb

```diff
@@ -2,7 +2,6 @@
 require "rspec/expectations"
```

Comment:
Remove this require "watir" as you no longer need it. Also use single quotes ' ' when ever possible. Because you can pass variables in with double quotes "#{something}" but you can't with single quotes, ruby runs faster with singles and thus it is consider best practice to use them by default.

Reply:
Got it, and updated!